aries_cloudagent/transport/inbound/ws.py

```diff
@@ -16,9 +16,9 @@
 class WsTransport(BaseInboundTransport):
     """Websockets Transport class."""
 
-    is_external = False
-
-    def __init__(self, host: str, port: int, create_session, **kwargs) -> None:
+    def __init__(
+        self, host: str, port: int, create_session, is_external=False, **kwargs
+    ) -> None:
```

Comment:
`is_external` is being defined here, but it's never used. It's not being passed to the parent class